docs/cosa-significa-essere-un-dev/README.md

```diff
@@ -0,0 +1,97 @@
+
+Solitamente, quando ci appassioniamo a qualcosa tendiamo a dedicare molto del nostro tempo a quella specifica attività; la coltiviamo negli anni prendendocene cura, approfondendo ogni aspetto e cercando di scoprire ogni singola caratteristica che possiede...in buona sostanza: _la rendiamo nostra_.
+
+Forti di questa consapevolezza possiamo quindi affermare che ciò che farà la differenza tra il nostro successo e la nostra sconfitta sarà la passione che metteremo in ciò che facciamo; se non avremo passione, difficilmente riusciremo ad accettare una sconfitta e a rialzarci più forti di prima; senza passione non riusciremo a trovare la forza di studiare, magari dopo una giornata di lavoro.
+
+Se però avremo passione, avremo dalla nostra una risorsa sorprendente: il dono della _Perseveranza_.
+
+Non importerà quindi quante volte proveremo, senza successo, a completare un'esercizio o ad imparare una tecnologia: avremo sempre la forza necessaria per incassare il colpo, prendere fiato, e provare a superare noi stessi.
+
+Il mio consiglio per te caro lettore, è quello di tenere a mente, per l'intera durata di questo manoscritto, il ruolo che ricopre la passione riguardo al percorso che hai scelto di intraprendere: se avrai passione, avrai la forza di perseverare.
+
+_Persevera con passione: a quel punto l'unico limite che avrai, sarai tu._
+
+## La curiosità: il potere della scoperta
+
+Qualità caratteristica dell'essere umano, la curiosità è l'impulso a scoprire, ricercare e comprendere ciò che ancora non conosciamo. Se cerchiamo la definizione di "curiosità" su Wikipedia infatti, scopriamo che:
+
+> _La curiosità è un aspetto emozionale che descrive un numero di meccanismi e comportamenti che hanno come fine il placare l'impulso degli esseri viventi a trarre informazioni ed interagire con l'ambiente._
+
+Il percorso che hai deciso di intraprendere pullula di informazioni che non aspettano altro che essere scoperte, studiate e messe in pratica; ci teniamo però a preparti: alcune nozioni potranno risultarti piuttosto semplici, altre dannatamente complesse.
+
+Nonostante ciò, non scoraggiarti; la nostra intenzione non è quella di spaventarti, ma di consigliarti e guidarti nel migliore dei modi; proprio per questo, ti inviatiamo a riflettere su un'aspetto tanto scontato quanto potente: il potere dell'informazione libera.
+
+Proprio come il libro che stai leggendo in questo momento, in rete esistono miliardi di informazioni a cui puoi avere accesso: aspettano solo di essere ricercate! Queste pagine che stai leggendo sono presenti sul web, e molto probabilmente ne sei venuto a conoscenza perché ti sei imbattuto in esse navigando per la rete, o magari perché chi ti ha consigliato questo libro lo ha cercato a sua volta. Poco importa, quello che è davvero importante è che tu prenda consapevolezza del fatto che il web pullula di informazioni a cui puoi accedere, e che l'unica cosa che ti separa da esse, indovina un po', _sei tu_!
```

Comment:
"Proprio come il libro che stai leggendo in questo momento" lo sostituirei con "Proprio come questo libro", perché dopo citi ancora "queste pagine che stai leggendo" e può risultare ripetitivo.